uninstall.zsh

```diff
@@ -34,3 +34,11 @@ else
   echo "✅ ${GREEN}~/.config/nvim${NC} is not symlinked"
 fi
 
+# Remove NvChad
```

Comment:
I added this but wasn't sure if if you wanted it. Lemme know if you'd rather me leave it out. If anything maybe just a message that says "The NvChad folder was not deleted" just so we are aware of everything installed and uninstalled.

Reply:
I'm good with this 👍